packages/tokens/config.js

```diff
@@ -62,4 +66,29 @@ EDSStyleDictionary.registerTransform({
   },
 });
 
+// copied from https://github.com/amzn/style-dictionary/blob/v3.0.0-rc.1/lib/common/formats.js#L96
+function minifyDictionary(obj) {
```

Comment:
maybe name `minifyCSSDictionary` to distinguish?

Reply:
I'll go with `minifyDictionaryToCSSVars`